global/scripts/FieldMapPreprocessingAll.sh

```diff
@@ -101,8 +101,8 @@ case $DistortionCorrection in
         # --------------------------------------
 
         MagnitudeInputName=`getopt1 "--fmapmag" $@`
-		    PhaseInputName=`getopt1 "--fmapphase" $@`
-		    DeltaTE=`getopt1 "--echodiff" $@`
+	    PhaseInputName=`getopt1 "--fmapphase" $@`
+		DeltaTE=`getopt1 "--echodiff" $@`
```

Comment:
In terms of style, these whitespace changes show that the file has a mix of tabs and spaces (even within a single line), and the current changes also leave edited lines inconsistent with their neighbors (these two changed lines use a tab and 4 spaces, and then two tabs for the next line).  I suggest trying to match the indentation style of at least consecutive lines within lines you were going to edit anyway.

Reply:
I changed this in my most recent commit. There were several lines in the FieldmapPreprocessingAll.sh script with such inconsistencies. I changed all the indentations in the file to 4 spaces, as this seemed as the most common pattern. There were also a few lines with 2-space indentation that I increased to 4 spaces.